lux/core/series.py

```diff
@@ -14,20 +14,26 @@
 
 import pandas as pd
 class LuxSeries(pd.Series):
-	# _metadata =  ['name','_intent','data_type_lookup','data_type',
-	# 			 'data_model_lookup','data_model','unique_values','cardinality',
-	# 			'min_max','plot_config', '_current_vis','_widget', '_recommendation']
+	_metadata = ['_intent','data_type_lookup','data_type',
+				 'data_model_lookup','data_model','unique_values','cardinality','_rec_info', '_pandas_only',
+				'_min_max','plot_config', '_current_vis','_widget', '_recommendation','_prev','_history', '_saved_export']
 	def __init__(self,*args, **kw):
 		super(LuxSeries, self).__init__(*args, **kw)
+
 	@property
 	def _constructor(self):
 		return LuxSeries
 
 	@property
 	def _constructor_expanddim(self):
 		from lux.core.frame import LuxDataFrame
```

Comment:
Is there a reason why we don't need to do __finalize__ here? I tested out the example [here](https://github.com/lux-org/lux/issues/65#issue-679200159) but the metadata did not propagate without the explicit `__finalize__`.